content/docs/guides/author_develop/pwa-amp/amp-as-pwa.md

```diff
@@ -0,0 +1,110 @@
+
+1. It has a [Web App Manifest](https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/), prompting the “Add to Homescreen” banner.
+1. It has a [Service Worker](https://developers.google.com/web/fundamentals/getting-started/primers/service-workers) and, therefore, allows offline access, among other things.
+
+When a user visits [AMPbyExample](http://ampbyexample.com/) from a AMP-supporting platform and then clicks continues the onward journey onto the same site, they navigate away from the AMP Cache to the origin. The website still uses the AMP library, of course, but because it now lives on the origin, it can use a service worker, can prompt to install and so on.
+
+{% call callout('Remember', type='caution') %}
+The Service Worker won't be able to interact with the AMP-cached version of your page. Use it for onward journeys to your origin.
+{% endcall %}
+
+## Add a Web App Manifest
+
+Adding a [Web App Manifest](https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/) to your AMP pages ensures that users can install your site to their devices' homescreen. There's nothing special about Web App manifests in AMP.
+
+First, create the manifest:
+
+[sourcecode:json]
+{
+  "short_name": "ABE",
+  "name": "AMPByExample",
+  "icons": [
+    {
+      "src": "launcher-icon-1x.png",
+      "type": "image/png",
+      "sizes": "48x48"
+    },
+    {
+      "src": "launcher-icon-2x.png",
+      "type": "image/png",
+      "sizes": "96x96"
+    },
+    {
+      "src": "launcher-icon-4x.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    }
+  ],
+  "start_url": "index.html?launcher=true"
+}
+[/sourcecode]
+
+Then link to it from the `<head>` of your AMP page:
+
+[sourcecode:html]
+<link rel="manifest" href="/manifest.json">
+[/sourcecode]
+
+{% call callout('Tip', type='success') %}
+Learn more about the [Web App Manifest at WebFundamentals](https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/).
+{% endcall %}
+
+## Install a Service Worker to enable offline access
+
+A Service Worker is a client-side proxy that sits between your page and your server, and can be used to build fantastic offline experiences, fast-loading app shell scenarios and the sending of push notifications.
+
+{% call callout('Note', type='note') %}
+If the concept of Service Workers is new to you, read the [introduction at WebFundamentals](https://developers.google.com/web/fundamentals/getting-started/primers/service-workers).
+{% endcall %}
+
+As Service Worker needs to be registered on a given page, or the browser won't find or run it. By default, this is done with the help of a [little bit of JavaScript](https://developers.google.com/web/fundamentals/instant-and-offline/service-worker/registration). On AMP Pages, you use the [`<amp-install-serviceworker>`](/docs/reference/components/dynamic/amp-install-serviceworker) component to achieve the same, like so:
```

Comment:
Also, they need more than just the html for the component, they need the required custom-element script